crates/bevy_text/src/text.rs

```diff
@@ -323,11 +328,38 @@ impl Default for TextFont {
         Self {
             font: Default::default(),
             font_size: 20.0,
+            line_height: LineHeight::default(),
             font_smoothing: Default::default(),
         }
     }
 }
 
+/// Specifies the height of each line of text for `Text` and `Text2d`
+///
+/// Default is 1.2x the font size
+#[derive(Debug, Clone, Copy, Reflect)]
+pub enum LineHeight {
+    /// Set line height to a specific number of pixels
+    Pixels(f32),
```

Comment:
We can't reuse `bevy_ui`'s `Val` here, but it would be nice to align the naming of the pixel size variant to `LineHeight::Px(f32)`, so that it's consistent with `Val::Px(f32)`